update/filter.go

```diff
@@ -89,3 +90,16 @@ func (f *LockedFilter) Filter(u ControllerUpdate) ControllerResult {
 	}
 	return ControllerResult{}
 }
+
+type IgnoreFilter struct {
+}
+
+func (f *IgnoreFilter) Filter(u ControllerUpdate) ControllerResult {
```

Comment:
I reckon this should be filtering based on the `Controller` field -- the `Resource`* field has the definition from a file, while we want to filter out workloads that are ignored in the cluster.

*I know I know, shifting terminology. Ugh